### PR TITLE
Run integration tests natively in CircleCI

### PR DESCRIPTION
**Pull request by saschagrunert**
This way we don't need docker any more. :)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -69,41 +71,26 @@ workflows:
           requires:
             - build
       - docs-validation
-      - ginkgo
       - git-validation
-      - integration:
-          requires:
-            - build
-            - build-test-binaries
+      - integration
```
Comment (TomSweeneyRedHat) [inline .circleci/config.yml]: Is this line valid?

**Comment by haircommander** [comment]
Lgtm, thanks for taking this @saschagrunert

File: .circleci/config.yml (hunk 1 above)
Comment (saschagrunert) [reply .circleci/config.yml]: Yes :) there are no parameters needed any more

**Pull request merged**